DateRangeBox: add code snippets and extended description #5043 (#5111)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxDateRangeBox/1 Configuration/invalidStartDateMessage.md

---
---
##### shortDescription

> Specifies a message displayed if the typed start date is not a valid date. 

Reviewer: ```suggestion
Specifies a message for invalid start date input. 
```
